src/components/wallet/earn/ChainTransfer/ChainTransfer.vue

```diff
@@ -339,32 +354,54 @@ export default class ChainTransfer extends Vue {
         let exportTxId
         this.exportState = TxState.started
 
-        switch (sourceChain) {
-            case 'X':
-                exportTxId = await wallet.exportFromXChain(
-                    amt,
-                    destinationChain as SDK.ExportChainsX,
-                    this.importFeeBN
-                )
-                break
-            case 'P':
-                exportTxId = await wallet.exportFromPChain(
-                    amt,
-                    destinationChain as SDK.ExportChainsP,
-                    this.importFeeBN
-                )
-                break
-            case 'C':
-                exportTxId = await wallet.exportFromCChain(
-                    amt,
-                    destinationChain as SDK.ExportChainsC,
-                    this.exportFeeBN
-                )
-                break
-        }
+        try {
+            switch (sourceChain) {
+                case 'X':
+                    exportTxId = await wallet.exportFromXChain(
+                        amt,
+                        destinationChain as CrossChainsX,
+                        this.importFeeBN
+                    )
+                    break
+                case 'P':
+                    exportTxId = await wallet.exportFromPChain(
+                        amt,
+                        destinationChain as CrossChainsP,
+                        this.importFeeBN
+                    )
+                    break
+                case 'C':
+                    exportTxId = await wallet.exportFromCChain(
+                        amt,
+                        destinationChain as CrossChainsC,
+                        this.exportFeeBN
+                    )
+                    break
+            }
 
-        this.exportId = exportTxId
-        this.waitExportStatus(exportTxId)
+            this.exportId = exportTxId
+            this.waitExportStatus(exportTxId)
+        } catch (e: any) {
+            if (e instanceof SignatureError) {
+                this.$store.dispatch('Notifications/add', {
```

Comment:
This line will crash. We need to import the notification helper from the global module

Reply:
i'll remove this for now because the chain transfer multisig is not implemented in the suite and has no relation to this pr